gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,6 @@
+#Sun Jul 23 21:27:04 MSK 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-rc-1-bin.zip
```

Stable release is available

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-all.zip
```